tests/test_qeventloop.py

```diff
@@ -589,8 +591,8 @@ def cb3():
 
         loop._add_reader(c_sock.fileno(), cb1)
 
-    clent_task = asyncio.ensure_future(client_coro())
-    server_task = asyncio.ensure_future(server_coro())
+    asyncio.ensure_future(client_coro())
+    asyncio.ensure_future(server_coro())
```

Comment:
This was introduced recently: https://github.com/CabbageDevelopment/qasync/pull/92/commits/1d0deb41379d36227e2544e594c461b434065ff4 and fixed win+python3.8 tests. I am not entierly sure why this happens, perhaps a bug in previous python version.

Reply:
Sry, make that python 3.9

Reply:
I added underscore to the variable name to make the linter happy. It still does store the value of the variable.